### ISSUE TITLE
Docs for `prefer-compact` missing an example using `_.compact`

### ISSUE THREAD

**Issue by alexilyaev** [wix/eslint-plugin-lodash#165]
Reading this:  
https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/prefer-compact.md

There's no example using `_.compact`.

**Comment by markalfred** [comment]
Seconded. It's not apparent to me how `_.compact` could be useful (or used, at all) for me here:

```js
_.filter(arr, item => item.property === 'foo')
```

Is this just a false positive? Does `_.compact` take a predicate argument that I'm not aware of? 🤔

**Comment by ganimomer** [comment]
You're right! Added an example.
